apps/web/components/pages/task/IssueCard.tsx

```diff
@@ -105,33 +141,25 @@ function CreateLinkedTask({
 				taskToId: parentTask.id,
 
 				organizationId: task.organizationId,
-				action: TaskRelatedIssuesRelationEnum.RELATES_TO,
+				action: $actionType.current.data.value,
 			}).catch(console.error);
 
 			loadTeamTasksData(false).finally(() => {
 				setLoading(false);
 				modal.closeModal();
 			});
 		},
-		[task, queryCall, loadTeamTasksData, modal]
+		[task, queryCall, loadTeamTasksData, modal, $actionType]
 	);
 
 	const linkedTasks = task.linkedIssues?.map((t) => t.taskFrom.id) || [];
-	const unlinkedTasks = tasks.filter(
-		(t) =>
-			// Remove current task
+	const unlinkedTasks = tasks.filter((t) => {
+		return (
 			t.id !== task.id &&
 			!linkedTasks.includes(t.id) &&
-			// If current task is Epic then filter Epics from the list
-			(task.issueType === 'Epic'
-				? t.issueType !== 'Epic'
-				: task.issueType === 'Story'
-				? // If current task is Story then filter Epics and Stories from the list
-				  t.issueType !== 'Epic' && t.issueType !== 'Story'
-				: t.issueType === 'Bug' ||
-				  t.issueType === 'Task' ||
-				  t.issueType === null)
-	);
+			!['Epic', 'Story'].includes(t.issueType)
```

Comment:
@paradoxe35 this logic is breaking. Please refactor it.

If current task is EPIC then STORY, TASK, BUG can be children
If current task is  STORY then TASK, BUG can be children
If current task is TASK|BUG then both can be children.